gym/lib/gym/generators/package_command_generator_xcode7.rb

```diff
@@ -98,6 +99,10 @@ def pkg_path
           path = files.min_by(&:length)
           Gym.cache[:pkg_path] = File.join(temporary_output_path, "#{Gym.config[:output_name]}.pkg")
           FileUtils.cp(path, Gym.cache[:pkg_path]) unless File.expand_path(path).casecmp(File.expand_path(Gym.cache[:pkg_path]).downcase).zero?
+        elsif appPath
+          # Try to find .app file in the output directory. This is used when macOS is set and .app is being generated.
+          Gym.cache[:pkg_path] = File.join(temporary_output_path, "#{Gym.config[:output_name]}.app")
```

Comment:
Minor semantic comment: here we're attributing a `.app` to something called `pkg_path`. Looks like re-using the same cache key as the pkg itself simplify things a bit here (and avoid other issues probably), so I'd be inclined to name it to be more generic (such as output_path, or binary_path, or something else that doesn't necessarily specify the file extension in its name).

Reply:
Thanks, done and done! I tested the last changes on my build and everything seems to work.

Reply:
Thanks!